### PR TITLE
Swig install with optional transpile

### PR DESCRIPTION
**Pull request by conorluddy**

### CODE REVIEW

File: packages/swig-install/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@gilt-tech/swig-install",
   "description": "Installs and organizes front-end assets.",
-  "version": "2.6.8",
+  "version": "2.6.9",
```
Comment (njprrogers) [inline packages/swig-install/package.json]: Let's make this a minor

**Comment by njprrogers** [comment]
I think we want to move a src and dest folder now that we are effectively transpiling everything in public JS. What we have now is not great i.e. our src folder is embedded with our dependencies. We can do that as a major bump next as it will require a change to folder structure (or name anyway).

File: packages/swig-install/lib/transpile-public.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    ];
+
+    swig.log.task('Transpiling installed packages in /public/js (excluding /public/js/src)');
+    swig.log.warn('You only need to do this for local IE11 development/fixes');
+
+    return gulp.src(paths)
+      .pipe(tap((file) => {
+        swig.log.success('', `Transpiling: ${path.basename(file.path).grey}`);
+      }))
+      .pipe(babel({
```
Comment (njprrogers) [inline packages/swig-install/lib/transpile-public.js]: this will transpile our src folder which we don't want.

**Comment by conorluddy** [comment]
Yeah it's a bit messy alright. You'd only use this option on occasion though when you want to fix IE11, and that in itself should be a relatively temporary solution.

File: packages/swig-install/lib/transpile-public.js (hunk 2 above)
Comment (conorluddy) [reply packages/swig-install/lib/transpile-public.js]: Nope, it won't:
```
const paths =  [
       './public/js/**/*.js',
       '!./public/js/src/**/*'
     ];
```

File: packages/swig-install/package.json (hunk 1 above)
Comment (conorluddy) [reply packages/swig-install/package.json]: Can do, but it's not going to affect any existing functionality - it just adds an extra standalone option. Up to you, chief :)

File: packages/swig-install/lib/transpile-public.js (hunk 2 above)
Comment (njprrogers) [reply packages/swig-install/lib/transpile-public.js]: Ah, nice! 🙇

File: packages/swig-install/package.json (hunk 1 above)
Comment (njprrogers) [reply packages/swig-install/package.json]: Minor === New Feature 😄

File: packages/swig-install/lib/transpile-public.js (hunk 2 above)
Comment (conorluddy) [reply packages/swig-install/lib/transpile-public.js]: I mean we could be even moar specific and just hit very specific directories – component, controller, tracking etc. I figured it'd be safe to hit the same ones that get wiped anyways by `swig install`

File: packages/swig-install/package.json (hunk 1 above)
Comment (Meesayen) [reply packages/swig-install/package.json]: the versioning is handled by lerna, don't increse the number manually. and yes, choose "minor" when asked by lerna.

File: packages/swig-install/lib/transpile-public.js (hunk 2 above)
Comment (njprrogers) [reply packages/swig-install/lib/transpile-public.js]: You're bang on there with just excluding src...that's the way to go 👍

File: packages/swig-install/package.json (hunk 1 above)
Comment (conorluddy) [reply packages/swig-install/package.json]: Yeah I suspected Lerna would do that automatically alright. Will revert package.json...

**Pull request merged**